vdr/types/common.go

```diff
@@ -136,31 +136,37 @@ type ResolveMetadata struct {
 // CompoundService is a service type that can be used as target for github.com/nuts-foundation/go-did/did/document.go#UnmarshalServiceEndpoint
 type CompoundService map[string]string
 
+// KeyUsage is a bitmask used for specifying for what purposes a key in a DID document can be used (a.k.a. Verification Method relationships).
+type KeyUsage int
+
+// Is returns whether the specified KeyUsage is enabled.
+func (k KeyUsage) Is(other KeyUsage) bool {
+	return k&other == other
+}
+
+const (
```

Comment:
you could have used the left shift operator, C style

Reply:
Nice